tests/geometry_test.cc

```diff
@@ -79,4 +94,19 @@ TEST_CASE("Geometry is checked for 3D case", "[geometry][3D]") {
       }
     }
   }
+
+  SECTION("Check angle between two vectors") {
+    // Make geometry
+    const auto geometry = std::make_unique<mpm::Geometry<Dim>>();
+
+    Eigen::Matrix<double, 3, 1> vector_a;
+    vector_a << 3, 0, 0;
```

Comment:
Could you use non-zero third dimension? Otherwise it is the same as 2D

Reply:
Oh, yes!